Sources/X509/X509BaseTypes/TimeCalculations.swift

```diff
@@ -167,7 +167,7 @@ extension Int64 {
         // Same for remdays, the loop only terminates if the number is smaller than at most 31.
         //
         // Note that, unlike struct tm, we return ordinal month numbers as well as days (i.e. 1 to 12).
-        // This fits us better when working with generalizedtime and friends.
+        // This fits us better when working with generalized time and friends.
```

Comment:
I dont think `generalizedtime` is a reference to something, right? So this must be a typo...

Reply:
It is, that's the ASN1 type.

Reply:
Probably should be uppercased though like this: [`GeneralizedTime`](https://en.wikipedia.org/wiki/GeneralizedTime)

Reply:
fixed